openpype/style/__init__.py

```diff
@@ -7,11 +7,28 @@
 
 from .color_defs import parse_color
 
+current_dir = os.path.dirname(os.path.abspath(__file__))
 
-_STYLESHEET_CACHE = None
-_FONT_IDS = None
 
-current_dir = os.path.dirname(os.path.abspath(__file__))
+# Default colors
+# - default color used in tool icons
+_TOOLS_ICON_COLOR = "#ffffff"
+# - entities icon color - use 'get_default_asset_icon_color'
+_DEFAULT_ENTITY_ICON_COLOR = "#fb9c15"
+# - disabled entitie
+_DISABLED_ENTITY_ICON_ICON_COLOR = "#808080"
```

Comment:
What's the reason we still need these globals if they are read from `openpype/style/data.json` by default? Shouldn't we choose one of the two instead of both?

It's added [here](https://github.com/pypeclub/OpenPype/pull/2889/files#diff-5b574387dd928faaa5b0ac039d1849c29f5cb781162b8b3a48024f83252bbb83R61-R63).

Would `data.json` ever not exist? And why wouldn't it?